### PR TITLE
blast: remove install workaround

### PR DESCRIPTION
**Pull request by iMichka**
The patch is already merged upstream and will be present
in release 2.6.0. This removes the need for the install
workaround.

### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### Updates to existing formula: have you

- [ ] Removed the `revision` line, if any, when bumping the version number?
- [x] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [x] Not altered the `bottle` section?
- [x] Checked that the tests still pass?

### CODE REVIEW

**Comment by iMichka** [comment]
Brew audit complains about:
> 'fails_with :llvm' is now a no-op so should be removed

@MikeMcQuaid This is due to that change in https://github.com/Homebrew/brew/pull/1673
The thing is, I have no clue what should be used to replace these lines :) Could you maybe give me a hint?

**Comment by jonchang** [comment]
You can just take the whole `fails_with :llvm` block out and it should work the same.

**Comment by iMichka** [comment]
Done. I just double-checked, 2335 was an older build (under 10.7); so it is no more supported anyway by homebrew. I was troubled because I was thinking: how can the formula fail with that llvm version if I remove that information. But if it is not supported anyway, removal is the way to go :). Thanks

**Comment by MikeMcQuaid** [comment]
> But if it is not supported anyway, removal is the way to go :). Thanks

Yep, we never use that compiler now.

**Comment by iMichka** [comment]
The build failures seem unrelated, some other formula could not build but for different reasons. I think this is good to go for now.

**Comment by jonchang** [comment]
@BrewTestBot test this please

File: blast.rb [hunk 1]
```diff
@@ -38,6 +33,12 @@ class Blast < Formula
   depends_on "pcre" => :recommended
   depends_on :python if MacOS.version <= :snow_leopard
 
+  patch do
+    # Fixed upstream in future version 2.6.0
+    url "https://gist.githubusercontent.com/iMichka/81ac6de72010c7812a47fc277dd21cb3/raw/80acfe731d2e4373b33f76b3da9aec2192b559b2/blast25.patch"
```
Comment (ilovezfs) [inline blast.rb]: Please open a PR to https://github.com/Homebrew/formula-patches with this :)

**Comment by iMichka** [comment]
Done. The build errors are unrelated (these are other packages failing to install due to missing dependencies).

**Comment by jonchang** [comment]
Thanks @iMichka!

**Pull request closed without merging**